blackbox/docs/sql/scalar.txt

```diff
@@ -215,6 +215,41 @@ locale-sensitive or context-sensitive mappings.
     SELECT 1 row in set (... sec)
 
 
+sha1('string') returns string
+-----------------------------
+
+Computes the SHA1 checksum of the given string.
+
+::
+
+    cr> select sha1('crate')
+    ... limit 1;
+    +------------------------------------------+
+    | sha1('crate')                            |
+    +------------------------------------------+
+    | 1673dc397042322a0a5ac49c79cc08d3a25cb0f6 |
+    +------------------------------------------+
+    SELECT 1 row in set (... sec)
+
+sha256('string') returns string
```

Comment:
I find it more convenient to have another function rather than parameterizing. If we want to comply with MySQL, then we should change the function.